libr/include/r_io.h

```diff
@@ -190,11 +190,29 @@ typedef struct r_io_map_t {
 	int fd;
 	int perm;
 	ut32 id;
+	ut64 ts;
 	RInterval itv;
-	ut64 delta; // paddr = itv.addr + delta
+	ut64 delta; // paddr = itv.addr + delta		//wtf no, this is not how it works
```

Comment:
how it works? i recall being confused when using this delta field some  time ago

Reply:
paddr = vaddr - itv.addr + delta

Reply:
we can fix this later